cluster/node.go

```diff
@@ -201,7 +202,8 @@ func (n HTTPNode) GetName() string {
 func handleResp(rsp *http.Response) ([]byte, error) {
 	defer rsp.Body.Close()
 	if rsp.StatusCode != 200 {
-		ioutil.ReadAll(rsp.Body)
+		// Read in body so that the connection can be reused
+		io.Copy(ioutil.Discard, rsp.Body)
```

Comment:
why didn't you do this for `rsp.StatusCode == 200` branch?

Reply:
This line of code reads in and *discards* everything in the response body so that the connection is clean for reuse. in the 200 case, we want that data!

Reply:
oh.  I thought maybe there would be a io.Copy call that could help the connection to be reused.